.github/workflows/dependency-review.yaml

```diff
@@ -71,6 +71,7 @@ jobs:
             Apache-2.0 AND BSD-3-Clause,
             Apache-2.0,
             BlueOak-1.0.0,
+            BSD,
```

Comment:
The package in question appears to be using 3-Clause BSD license, but it's package.json states just "BSD" which is insufficient.

* https://github.com/javiertury/babel-plugin-transform-import-meta/blob/5db4386fbde67894d6d7d9bda61530fe75d7869a/LICENSE
* https://github.com/javiertury/babel-plugin-transform-import-meta/blob/5db4386fbde67894d6d7d9bda61530fe75d7869a/package.json#L9

Reply:
I made a PR to the project to update the license. https://github.com/javiertury/babel-plugin-transform-import-meta/pull/23